l1-contracts/test/foundry/integration/deploy-scripts/script/DeployL1.s.sol

```diff
@@ -0,0 +1,695 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+// solhint-disable no-console
+
+import {Script, console2 as console} from "forge-std/Script.sol";
+import {stdToml} from "forge-std/StdToml.sol";
+import {ProxyAdmin} from "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
+import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+
+import {Utils} from "./Utils.sol";
+import {Multicall3} from "contracts/dev-contracts/Multicall3.sol";
+import {VerifierParams, IVerifier} from "contracts/state-transition/chain-interfaces/IVerifier.sol";
+import {DefaultUpgrade} from "contracts/upgrades/DefaultUpgrade.sol";
+import {Governance} from "contracts/governance/Governance.sol";
+import {GenesisUpgrade} from "contracts/upgrades/GenesisUpgrade.sol";
+import {ValidatorTimelock} from "contracts/state-transition/ValidatorTimelock.sol";
+import {Bridgehub} from "contracts/bridgehub/Bridgehub.sol";
+import {ExecutorFacet} from "contracts/state-transition/chain-deps/facets/Executor.sol";
+import {AdminFacet} from "contracts/state-transition/chain-deps/facets/Admin.sol";
+import {MailboxFacet} from "contracts/state-transition/chain-deps/facets/Mailbox.sol";
+import {GettersFacet} from "contracts/state-transition/chain-deps/facets/Getters.sol";
+import {DiamondInit} from "contracts/state-transition/chain-deps/DiamondInit.sol";
+import {StateTransitionManager} from "contracts/state-transition/StateTransitionManager.sol";
+import {StateTransitionManagerInitializeData} from "contracts/state-transition/IStateTransitionManager.sol";
+import {IStateTransitionManager} from "contracts/state-transition/IStateTransitionManager.sol";
+import {Diamond} from "contracts/state-transition/libraries/Diamond.sol";
+import {InitializeDataNewChain as DiamondInitializeDataNewChain} from "contracts/state-transition/chain-interfaces/IDiamondInit.sol";
+import {FeeParams, PubdataPricingMode} from "contracts/state-transition/chain-deps/ZkSyncHyperchainStorage.sol";
+import {L1SharedBridge} from "contracts/bridge/L1SharedBridge.sol";
+import {L1ERC20Bridge} from "contracts/bridge/L1ERC20Bridge.sol";
+import {DiamondProxy} from "contracts/state-transition/chain-deps/DiamondProxy.sol";
+
+contract DeployL1Script is Script {
```

Comment:
Can we maybe import this contract directly from the repo to not copy-pasting it?